### DISCUSSION TITLE
项目打包后resources文件下面的静态图片读取不到了

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#105] [Q&A]
### 涉及的编程语言

Java

### 疑问描述

在开发的时候静态图片是放在resources下面的 通过：
PathResource resource = Resource.of(Paths.get("image.png"));
    ResourceImage resourceImage = Image.of(resource);
    event.getFriend().sendBlocking(resourceImage);
来发送图片是正常的

但是在部署环境的时候他说找不到文件

**Comment by ForteScarlet** [comment]
Java中的Path读取的是文件，resources中的内容打包到jar里面后不再属于系统文件读不到很正常。
请自行通过搜索引擎了解它们的区别以及如何读取jar中的资源文件。
